docs/Reference/README.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: "Reference"
+---
+
+These resources act as references that provide technical descriptions of F'. You will find API documentation here.
+
+## API Documentation
+1. [GDS CLI Design](./fprime/UsersGuide/dev/gds-cli-dev.html)
```

Comment:
This first link needs to be relative and point to the markdown file instead of HTML, e.g. `../UsersGuide/dev/gds-cli-dev.md`

Reply:
Same for `[CMake User API](./fprime/UsersGuide/cmake/cmake-api.html)`

Reply:
Thanks, I believe I fixed these ones but I noticed there are a few other links that are broken right now as well. I noticed that the one going from Getting started page to the installing page is broken as well. I will check because I saw more broken links in the test and fix those as well.